dali/operators/image/color/color_twist.cc

```diff
@@ -24,51 +24,62 @@ using TheKernel = kernels::LinearTransformationCpu<Out, In, 3, 3, 3>;
 }  // namespace
 
 DALI_SCHEMA(Hsv)
-    .DocStr(R"code(This operator performs HSV manipulation.
-To change hue, saturation and/or value of the image, pass corresponding coefficients.
-Keep in mind, that `hue` has additive delta argument,
-while for `saturation` and `value` they are multiplicative.
+    .DocStr(R"code(Adjusts hue, saturation and value (brightness) for the images.
 
-This operator accepts RGB color space as an input.
+To change the hue, the saturation, and/or the value of the image, pass the corresponding
+coefficients. Remember that the ``hue`` is an additive delta argument,
+while for ``saturation`` and ``value``, the arguments are multiplicative.
 
-For performance reasons, the operation is approximated by a linear transform in RGB space.
+This operator accepts images in the RGB color space.
+
+For performance reasons, the operation is approximated by a linear transform in the RGB space.
 The color vector is projected along the neutral (gray) axis,
```

Comment:
This is mathematically incorrect. Currently I don't know how to write it better, but this is simply not true. Projecting the vector along this axis would lose any information about the brightness.

Reply:
So what do you suggest?